functions/src/lobby.ts

```diff
@@ -97,6 +97,14 @@ export const onLobbyUpdate = functions.firestore.document("/lobbies/{code}").onU
   if (lobby.state == "PROMPT" && oldLobby.state != "PROMPT") {
     await startPrompt(lobbyDocRef);
   }
+
+  if (lobby.state == "END" && oldLobby.state != "END") {
+    addRole(lobbyDocRef);
+  }
```

Comment:
what sets the state to `END`?

Reply:
I'm assuming voting will have a function that will change the lobby state  right?

Reply:
Hmm you can @cmcclusky should talk and make sure you aren't each assuming that eachother is going to do that

Reply:
For sure, I'll message him